src/operators/add.ml

```diff
@@ -212,7 +213,7 @@ let () =
          ~kind ~renorm ~power
          (List.map2 (fun w s -> (w, s)) weights sources)
          (fun _ -> ())
-         (fun _ buf tmp -> Video.Image.add tmp buf)
+         (fun _ buf tmp -> Video.Canvas.Image.add tmp buf)
```

Comment:
Do you plan on changing this do use canvas?

Reply:
Thanks for spotting this, it was a mistake...